rep-0003.rst

```diff
@@ -180,6 +179,48 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+
+  - Some C++11 will be enabled (TODO(esteve): need to determine which ones)
```

Comment:
If they are "enabled" I assume that implies that code can use it which automatically makes it a requirement. Based on that the minimum requirement is not C++03 anymore but C++11. And then I am not sure why we should artificially restrict the feature set?